### PR TITLE
(CI) Enable NLS for macOS builds

### PR DESCRIPTION
**Pull request by csparker247**
When developing #687, it was noted that NLS isn't enabled on the macOS Travis CI builds. 
This PR:
1) Installs `gettext` via Homebrew
2) Removes the `EXIV2_ENABLE_NLS=OFF` CMake option
3) Since `gettext` is keg-only, it sets the `CMAKE_PREFIX_PATH` to the `gettext` install location

### CODE REVIEW

**Comment by csparker247** [comment]
Build error is because of issue fixed in #687. Test should be rerun after that PR makes it into `master`.

File: ci/run.sh
Comment (D4N) [inline ci/run.sh]: Is manually adding the prefix necessary? (This is not a nitpick, I have no clue about OSX, I would have assumed that cmake would pick this up)

**Comment by D4N** [comment]
This is blocked by #690.

@csparker247 Thanks for creating this! Once #690 is merged, could you rebase this on top of master? (i.e. `git pull --rebase origin master`). And just a tiny nitpick: could you please change the curl braces in the commit message to `[]` for consistency?

File: ci/run.sh
Comment (piponazo) [reply ci/run.sh]: Yes, sometimes it is needed to do this kind of things. For example when working with Qt, it is also needed to help CMake to find external libraries. I think this is fine.

**Review by piponazo — DISMISSED** [review]

File: ci/run.sh
Comment (csparker247) [reply ci/run.sh]: Yeah, this is more of a Homebrew issue than an OSX thing. They put certain packages in non-standard places so that they don't conflict with system-provided things. It makes Mac development unwieldy sometimes.

**Review by piponazo — APPROVED** [review]

**Pull request merged**
